### PR TITLE
Tag NearestNeighbors.jl v0.2.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [KristofferC/NearestNeighbors.jl](https://github.com/KristofferC/NearestNeighbors.jl)
Release: [v0.2.0](https://github.com/KristofferC/NearestNeighbors.jl/releases/tag/v0.2.0)
Travis: [Travis Build Status](https://travis-ci.org/KristofferC/NearestNeighbors.jl/builds/204897382)
Diff: [vs v0.1.1](https://github.com/KristofferC/NearestNeighbors.jl/compare/88368b584faed8850688767ed807882037d12963...eb05a1a17fd0604db62ee64e35f8a08cfc284caf)
`requires` vs v0.1.1: 
```diff
--- 0.1.1/requires
+++ 0.2.0/requires
@@ -1,4 +1,4 @@
-julia 0.5-
-Distances 0.3 0.4
+julia 0.5
+Distances 0.4
 StaticArrays 0.0.4
-Compat 0.8.4
+Compat 0.18.0
```
cc: @KristofferC

### CODE REVIEW

File: NearestNeighbors/versions/0.2.0/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+julia 0.5
+Distances 0.4
+StaticArrays 0.0.4
+Compat 0.18.0
```
Comment (tkelman) [inline NearestNeighbors/versions/0.2.0/requires]: 0.17 for type aliases that aren't parametric

Comment (KristofferC) [inline NearestNeighbors/versions/0.2.0/requires]: Iäm fine with requiring 0.18.

**Comment by tkelman** [comment]
re-trigger that? travis is mia again

**Comment by KristofferC** [comment]
shouldnt have done that...

**Pull request closed without merging**